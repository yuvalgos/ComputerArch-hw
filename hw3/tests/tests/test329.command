./dflow_calc tests/test329.dat tests/test329.in p28 d128 d69 d57 d8 d3 p22 p87 p59 p39 d67 p198 d130 p83 d111 p125 p177 p90 d95 d18 d57 p79 p170 p13 p69 d52 p178 p52 p54 p108 p127 d128 p190 d48 p52 d151 p33 d140 p194 p56 p39 d190 d63 p123 p6 p40 d110 d179 d133 p164 d183 d99 p1 d172 d37 p81 p192 d33 p110 p84 p151 p146 p90 d136 p128 p136 p49 p17 p90 d43 d124 p24 d64 p139