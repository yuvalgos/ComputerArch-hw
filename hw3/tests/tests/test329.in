7 6 19 3
2 5 1 4
4 14 19 23
2 0 11 2
3 2 1 4
5 17 15 2
4 1 5 4
8 8 0 1
2 14 23 19
7 30 5 10
6 17 23 5
4 15 17 8
6 11 8 19
0 9 17 1
5 12 5 18
0 9 23 2
5 20 4 23
3 22 22 27
3 4 2 30
1 0 18 8
3 23 5 19
8 12 0 19
7 25 27 8
7 3 12 0
4 9 11 12
3 5 2 13
3 13 1 27
1 28 20 24
5 16 12 8
8 17 26 0
6 17 9 15
2 28 14 12
6 13 2 13
6 13 7 2
6 0 8 29
2 17 25 4
1 28 25 10
5 3 6 20
2 30 24 10
1 18 25 23
4 12 30 5
4 3 2 22
1 22 16 18
6 12 9 27
1 1 5 10
1 1 8 2
1 4 3 27
6 9 25 29
5 29 13 2
2 18 27 20
7 12 4 17
5 12 28 0
8 16 14 3
5 23 21 0
3 2 30 12
6 4 4 18
5 15 16 4
6 4 25 1
7 25 5 9
5 9 8 26
4 30 23 20
3 14 29 8
0 4 22 17
6 6 0 1
0 29 6 6
3 6 7 21
7 17 20 30
3 17 15 21
1 28 4 30
4 26 11 0
1 27 26 20
2 4 19 23
3 27 3 13
3 13 9 10
5 4 30 30
5 22 26 9
5 21 20 29
5 25 11 23
1 18 31 11
5 3 4 30
8 29 18 24
0 14 27 2
4 20 18 28
4 3 9 29
7 18 30 21
2 15 15 29
6 5 18 11
3 6 13 25
3 11 14 4
5 29 14 1
0 19 6 26
3 10 20 27
2 12 13 29
6 9 25 11
2 27 12 25
3 12 18 7
4 11 4 7
1 20 8 28
6 18 18 2
3 11 30 6
8 10 28 14
3 22 28 22
8 27 24 10
4 25 30 20
8 28 3 17
1 22 27 5
3 23 4 27
0 25 28 19
3 24 30 24
5 5 20 18
3 2 26 0
4 12 22 2
3 0 20 26
0 31 25 13
5 28 17 5
4 7 2 15
0 26 21 7
8 7 29 7
8 14 7 11
5 13 15 26
0 21 20 4
6 22 23 15
0 8 19 22
2 27 18 26
5 11 24 22
2 10 12 11
0 27 14 12
3 21 26 11
3 9 5 29
2 8 13 26
4 4 17 5
1 3 25 16
4 4 12 24
5 6 30 8
8 2 31 20
1 25 0 31
2 27 18 29
1 22 19 18
6 18 7 24
8 24 26 30
1 2 31 29
3 3 26 28
5 0 21 11
1 11 31 31
6 8 13 12
0 21 13 14
5 31 3 14
1 19 17 28
7 19 26 8
7 5 2 24
1 27 12 0
6 20 28 21
7 27 2 22
8 3 7 30
1 21 30 30
1 24 20 9
6 3 3 23
3 8 31 8
2 26 24 0
6 12 7 28
4 11 30 5
3 5 18 13
2 27 31 18
1 15 4 27
8 8 21 6
4 11 6 7
1 5 18 0
8 1 4 25
8 25 24 28
2 26 11 13
8 4 17 19
5 11 1 24
8 13 19 19
1 31 2 27
1 29 14 31
7 10 10 22
1 19 31 7
5 12 13 30
1 0 19 23
3 17 5 26
8 9 12 17
2 3 2 18
1 28 6 5
8 10 8 2
5 27 0 26
3 13 28 24
7 4 31 30
4 13 5 24
4 17 3 12
4 12 18 30
8 13 28 0
6 23 18 18
7 28 11 15
1 11 8 7
3 27 25 21
8 5 30 15
0 10 31 13
8 26 21 3
6 8 23 13
7 23 7 24
