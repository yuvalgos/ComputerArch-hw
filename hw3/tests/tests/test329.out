Reading the opcodes latency info from tests/test329.dat ... Got latency for 9 opcodes
Reading the program file tests/test329.in ... Found 200 instructions
getProgDepth()==241
getDepDepth(28)==65
getInstDeps(128)=={109,89}
getInstDeps(69)=={12,34}
getInstDeps(57)=={22,45}
getInstDeps(8)=={-1,-1}
getInstDeps(3)=={-1,-1}
getDepDepth(22)==39
getDepDepth(87)==82
getDepDepth(59)==39
getDepDepth(39)==58
getInstDeps(67)=={56,-1}
getDepDepth(198)==218
getInstDeps(130)=={67,109}
getDepDepth(83)==82
getInstDeps(111)=={105,110}
getDepDepth(125)==169
getDepDepth(177)==205
getDepDepth(90)==89
getInstDeps(95)=={84,-1}
getInstDeps(18)=={4,9}
getInstDeps(57)=={22,45}
getDepDepth(79)==93
getDepDepth(170)==182
getDepDepth(13)==22
getDepDepth(69)==56
getInstDeps(52)=={8,41}
getDepDepth(178)==181
getDepDepth(52)==18
getDepDepth(54)==74
getDepDepth(108)==80
getDepDepth(127)==169
getInstDeps(128)=={109,89}
getDepDepth(190)==195
getInstDeps(48)=={33,4}
getDepDepth(52)==18
getInstDeps(151)=={114,145}
getDepDepth(33)==7
getInstDeps(140)=={113,89}
getDepDepth(194)==193
getDepDepth(56)==41
getDepDepth(39)==58
getInstDeps(190)=={182,178}
getInstDeps(63)=={34,45}
getDepDepth(123)==124
getDepDepth(6)==5
getDepDepth(40)==14
getInstDeps(110)=={69,34}
getInstDeps(179)=={166,169}
getInstDeps(133)=={60,129}
getDepDepth(164)==131
getInstDeps(183)=={164,140}
getInstDeps(99)=={60,87}
getDepDepth(1)==0
getInstDeps(172)=={148,148}
getInstDeps(37)=={0,16}
getDepDepth(81)==81
getDepDepth(192)==193
getInstDeps(33)=={-1,4}
getDepDepth(110)==67
getDepDepth(84)==88
getDepDepth(151)==150
getDepDepth(146)==157
getDepDepth(90)==89
getInstDeps(136)=={98,89}
getDepDepth(128)==137
getDepDepth(136)==124
getDepDepth(49)==13
getDepDepth(17)==0
getDepDepth(90)==89
getInstDeps(43)=={24,-1}
getInstDeps(124)=={108,121}
getDepDepth(24)==65
getInstDeps(64)=={63,63}
getDepDepth(139)==100
